docs/setup/jetson.md

```diff
@@ -113,6 +113,11 @@ The recommended way to run ROS2 on Jetson is to use pre-built Docker container i
 3. Set the default network interface by setting Cyclone DDS configuration.
 
         export CYCLONEDDS_URI='<CycloneDDS><Domain><General><NetworkInterfaceAddress>l4tbr0</NetworkInterfaceAddress></General></Domain></CycloneDDS>'
+    !!! attention
+        **If you are using CycloneDDS (Galactic default) and want the Jetson to talk to the robot over USB and a laptop via Wi-Fi, you will need to take extra steps to use multiple interfaces.**
```

Comment:
"to use multiple interfaces" --> "to setup CycloneDDS to use multiple network interfaces"